### DISCUSSION TITLE
Domain is pending verification

### DISCUSSION THREAD

**Discussion by steveoni** [vercel/community#3883] [Help] [answered]
### Summary

Vercel complains about `Domain is pending verification`. we want to add a domain `data.hsl.harvard.edu` to our vercel app, but vercel complains that the domain is used by another vercel account and it provides a step to add some configuration to the root domain `harvard.edu` but we don't have access to it. 

We've tried different approaches to fix this but none is valid. We would love to know if there is a possible solution that does not involve updating the TXT in the root domain.

### Steps to Reproduce

<img width="950" alt="Screenshot 2023-09-01 at 14 53 20" src="https://github.com/vercel/community/assets/20904032/3da5e1e0-df0e-4cc7-a1b3-20dd9b6b83db">

**Comment by amyegan** [comment, marked as answer]
Hey @steveoni. We need to verify the apex domain to be sure of ownership. Please work with your domain administrator to add the DNS record for this 🙏

**Reply by hussein-hijazi856** [reply]
Is there a way to verify the domain through terraform? I mean I created the project domain through terraform, I need a way to get the verification TXT record through terraform, so that I can create it automatically. @amyegan
